### PR TITLE
Index Page

### PR DESCRIPTION
**Pull request by ImArnoldDCODES**
Still working on it...

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -7,6 +7,9 @@
   </head>
   <body>
     <div id="root"></div>
+    <!-- <div class="loader" id="app">
```
Comment (struckchure) [inline web/index.html:10]: Uhh, there's never a reason to edit this file this way

File: web/package.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "name": "share-seller",
   "version": "0.0.0",
   "scripts": {
+    "start": "vite start",
```
Comment (struckchure) [inline web/package.json:5]: and this is unnecessary, use the dev script instead

File: web/package.json [hunk 3]
```diff
@@ -12,6 +13,7 @@
     "postcss-import": "^14.0.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-preloaders": "^3.0.3",
```
Comment (struckchure) [inline web/package.json:16]: Nope, we would rather user a customer preload

**Review by struckchure — APPROVED** [review]
Nice Job @ImArnoldDCODES !!!!!!!!

File: web/index.html (hunk 1 above)
Comment (ImArnoldDCODES) [reply web/index.html:10]: now be me o

File: web/package.json (hunk 3 above)
Comment (ImArnoldDCODES) [reply web/package.json:16]: ok

File: web/index.html (hunk 1 above)
Comment (struckchure) [reply web/index.html:10]: I had that feeling :laughing:

**Pull request merged**